src/main/java/org/ambraproject/wombat/controller/FigurePageController.java

```diff
@@ -101,7 +102,7 @@ public String renderFigurePage(Model model, @SiteParam Site site,
 
     Map<String, ?> figureMetadata = articleMetadata.getFigureView().stream()
         .filter((Map<String, ?> fig) -> fig.get("doi").equals(assetPointer.getAssetDoi()))
-        .findAny().orElseThrow(RuntimeException::new);
+        .collect(MoreCollectors.onlyElement());
```

Comment:
This is unrelated, but I used `MoreCollectors.onlyElement()` elsewhere and figured that was as good an excuse as any to tighten up other places where we ought to use it.

Using `.findAny().orElseThrow(RuntimeException::new)` expects 1 element, throws an error on 0, and silently ignores extra elements in case there are more than 1. Using `.collect(MoreCollectors.onlyElement())` throws an error if there are fewer *or more* than 1.